intel_extension_for_deepspeed/xpu_accelerator.py

```diff
@@ -10,12 +10,12 @@ def __init__(self):
         self._communication_backend_name = 'ccl'
         def _check_and_mapping_mpich_env():
             import os
+            if  "RANK" not in os.environ and "PMIX_RANK" in os.environ:
+                os.environ['RANK'] = os.environ.get('PMIX_RANK')
+                print("mapping environment variable PMIX_RANK to RANK")
             if  "RANK" not in os.environ and "PMI_RANK" in os.environ:
                 os.environ['RANK'] = os.environ.get('PMI_RANK')
                 print("mapping environment variable PMI_RANK to RANK")
-            if  "WORLD_SIZE" not in os.environ and "PMI_SIZE" in os.environ:
```

Comment:
it is not required by borealis, but required by boris, and so may be required by the same launcher as in boris for feature customers .  imo, let's keep the code?

Reply:
Nope, this is a previously unnecessary part. Since we will export WORLD_SIZE=xx in our running script, there is no need for this.

Reply:
DeepSpeed comm need RANK, WORLD_SIZE, LOCAL_RANK to work properly without invoking mpi4py.   Do we also have LOCAL_RANK mapped?   I think this code should stay here for roubustness, if broder user still have a chance to hit this path.

Reply:
For world_size, I could keep it but with pmix launcher we won't have similar world_size in env like pmi_size. and for local_rank, yes there is and I will add it.